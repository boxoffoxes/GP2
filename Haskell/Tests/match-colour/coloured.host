[
	(n1, empty # grey)
	(n2, empty # blue)
|
	(e1, n1, n2, empty # dashed)
]
