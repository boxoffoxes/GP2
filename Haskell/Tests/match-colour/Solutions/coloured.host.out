[
 (n1, empty # red)
 (n2, empty)
|
 (e2, n2, n1, empty)
]
